pkg/scheduler/scheduler.go

```diff
@@ -776,7 +787,7 @@ func (s *Scheduler) reconcileEstimatorConnection(key util.QueueKey) error {
 		return nil
 	}
 
-	return estimatorclient.EstablishConnection(s.KubeClient, name, s.schedulerEstimatorCache, s.schedulerEstimatorServicePrefix, s.schedulerEstimatorClientConfig)
+	return estimatorclient.EstablishConnection(s.KubeClient, name, s.schedulerEstimatorCache, s.schedulerEstimatorServiceNamespace, s.schedulerEstimatorServicePrefix, s.schedulerEstimatorClientConfig)
```

Comment:
With this parameter, the number of parameters exceeds 5, I remember some tools might check this and would bring a warning for this.

This is not a blocker, do you have any idea how to improve it?

Reply:
Good point. Will push PR to address in a few.